test/test_util.py

```diff
@@ -0,0 +1,46 @@
+"""Test util."""
+
+from alpenhorn_chime import util
+from alpenhorn.acquisition import AcqType, FileType, AcqFileTypes
+
+
+def test_update_types(tables):
+    """Test update_types()."""
+
+    # Create the type data
+    util.update_types()
+
+    # Check
+    type_data = {
+        "corr": {"acq_class": "CorrAcqInfo", "file_class": "CorrFileInfo"},
+        "hfb": {"acq_class": "HFBAcqInfo", "file_class": "HFBFileInfo"},
+        "weather": {"acq_class": "WeatherAcqDetect", "file_class": "WeatherFileInfo"},
+        "rawadc": {"acq_class": "RawadcAcqInfo", "file_class": "RawadcFileInfo"},
+    }
+
+    for name, data in type_data.items():
+        at = AcqType.get(name=name)
+        assert at.info_class == "alpenhorn_chime.info." + data["acq_class"]
+
+        ft = FileType.get(name=name)
+        assert ft.info_class == "alpenhorn_chime.info." + data["file_class"]
+
+        fts = list(AcqFileTypes.select().where(AcqFileTypes.acq_type == at))
+        assert len(fts) == 1
+        assert fts[0].file_type == ft
+
+    # The calibration types are different
+    ft = FileType.get(name="calibration")
+    assert ft.info_class == "=alpenhorn_chime.info.cal_info_class"
```

Comment:
I presume this is a typo?
```suggestion
    assert ft.info_class == "alpenhorn_chime.info.cal_info_class"
```

Reply:
No, this is needed.

This is part of the CHIME-specific shennanigans: instead of being an info class itself, `alpenhorn_chime.info.cal_info_class` is a function which is passed the _acqtype_ and returns a _file_ info class.

The leading `=` is used to tell alpenhorn this is what's going on. (I could change the `=` to another character, if desired).

Reply:
documented here: https://github.com/radiocosmology/alpenhorn/blob/592ea09e948bcb6f3433aa30723160544097159b/alpenhorn/acquisition.py#L40

Part of rewrite part six: https://github.com/radiocosmology/alpenhorn/pull/149